unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Clipboard/ClipboardReadPromise.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using UnityEngine;
+
+public class ClipboardReadPromise : CustomYieldInstruction
```

Comment:
Name and scope of this class doesn't seem appropiate. I propose a rename and to move it to `CoroutineHelpers`, `Utils` or such.
```suggestion
public class Promise : CustomYieldInstruction
```

Reply:
Do you mean creating a Generic Promise? that's a good idea